ext/duckdb/column.c

```diff
@@ -87,5 +115,6 @@ void rbduckdb_init_duckdb_column(void) {
     rb_define_alloc_func(cDuckDBColumn, allocate);
 
     rb_define_private_method(cDuckDBColumn, "_type", duckdb_column__type, 0);
+    rb_define_method(cDuckDBColumn, "_logical_type", duckdb_column__logical_type, 0);
```

Comment:
Can we define "logical_type" method directory instead?
```suggestion
    rb_define_method(cDuckDBColumn, "logical_type", duckdb_column__logical_type, 0);
```

Reply:
fix: [1aff397](https://github.com/suketa/ruby-duckdb/pull/842/commits/1aff397adca374f1e1872019d1a5a0c46afc334a) I agree with you. We don't have to do that, although I thought I should use the same name with a `duckdb_column__logical_type` wrapper method for consistency.